application/ui/src/app/services/users.service.ts

```diff
@@ -150,7 +194,7 @@ export class UsersService {
         return organization;
       }),
       switchMap((organization) =>
-        this.http.post(`${this.urlApi}/email-matches`, { matches, organization })
+        this.http.post(`${this.urlApi}/email-matches`, { matches, organization,time,frequency })
```

Comment:
not sure to understand here it cannot be frequency, i believe it is duration you wanted to say